src/FormElement.php

```diff
@@ -677,7 +677,7 @@ public function toArray() {
    */
   protected function addControlsToArray(array &$output) {
     foreach ($this->controls as $name => $value) {
-      $has_property_interface = has_interface($value, 'FormPropertyInterface');
+      $has_property_interface = has_interface($value, 'Drupal\objective_forms\FormPropertyInterface');
```

Comment:
I... hmm... how does it get access to the `has_interface()` function? As in... it's in the [ReflectionHelpers.inc](https://github.com/discoverygarden/php_lib/blob/f62dc0550d545eaa584b09a6a384b1d354934ceb/ReflectionHelpers.inc#L61-L73) file, but we don't load that file here... looks like the `ReadOnlyProtectedMembers` bit includes it? Seems spooky... anyway.

_Could_... change to a use statement on the interface, and an `instanceof`... getting rid of a use of php_lib... novel? Shrug?